megamek/src/megamek/client/bot/princess/FireControl.java

```diff
@@ -1843,6 +1835,13 @@ FiringPlan getFullFiringPlan(final Entity shooter,
 
         // cycle through my weapons
         for (final Mounted weapon : shooter.getWeaponList()) {
+        	// respect restriction on manual AMS firing.
+        	if(!game.getOptions().booleanOption(OptionsConstants.ADVCOMBAT_TACOPS_MANUAL_AMS) &&
```

Comment:
Technically unrelated, but I noticed the bot was lighting off AMS systems at my mechs even though I didn't have "AMS as a weapon" on. So now the bot will respect that setting.